networkmanager/networkmanager.go

```diff
@@ -651,7 +654,9 @@ func (w *NMWrapper) activateConnection(ctx context.Context, ssid string) error {
 	}
 
 	nw.lastTried = now
-	w.lastSSID = ssid
+	if ssid != w.hotspotSSID {
```

Comment:
not sure why this is necessary

Reply:
We want to only record the last actual wifi connection attempted, and thus get any errors that happened with it. Without this, the hotspot itself (when it starts up) would always be the "lastSSID" and thus we couldn't fetch errors for the actual network tried.